### PR TITLE
getPreferredEpoch case 4.5 - subset membership

### PR DESCRIPTION
**Pull request by mixmix**
fixes https://github.com/ssbc/ssb-tribes2/issues/113

### CODE REVIEW

File: lib/epochs.js [hunk 1]
```diff
@@ -544,13 +545,18 @@ function buildPreferredEpoch(reduce, cb) {
...
-      return cb(
-        Error('Tips do not have same membership - case not handled yet')
-      )
+    } else if (isSubsetOf(members1, members2)) {
+      preferredEpoch = tips[0]
+    } else if (
+      isSubsetOf(members2, members1)
+    ) {
+      preferredEpoch = tips[1]
     }
```
Comment (Powersource) [inline lib/epochs.js:557]: could be nice with some comments here or?

**Review by Powersource — APPROVED** [review]
wow that's simple and clean.

clarifying comments could be nice

File: lib/epochs.js (hunk 1 above)
Comment (mixmix) [reply lib/epochs.js:557]: done

Comment (Powersource) [reply lib/epochs.js:557]: mm where?

**Pull request merged**